### PR TITLE
add css + related song components with tooltip functionality 

### PR DESCRIPTION
**Pull request by shiju-an**
- add related song components 
-- related songs list
-- related song
-- artist tooltip

- add functionality to artist tooltip
-- rendering with correct artist data and image

- add basic css to related songs list refactor later on using css modules

- work on tests - need to backtrack and write tests for artist tooltip functionality

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -1,32 +1,63 @@
...
+    
     render() {
         return (
+        <div className="content">
         <div>
-            <div>TESTING REACT</div>
-        </div>
-        )
-    }
+            <p><ArtistTooltip contents={<RelatedSongsList relatedSongs={this.state.relatedSongs} />} position={'bottom'}>Artist Name Test</ArtistTooltip></p>
```
Comment (mgalfano) [inline client/src/App.jsx:45]: Is closing tag of ArtistTooltip necessary? I think you can just close the tag with a forward slash, unless it breaks the p tag... in which case, please ignore:
```suggestion
            <p><ArtistTooltip contents={<RelatedSongsList relatedSongs={this.state.relatedSongs} />} position={'bottom'} />Artist Name Test</p>
```

File: client/src/RelatedSong.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+        return (
+            <div>
+                <div className="songContainer">
+                    <img src={this.props.relatedSong.song_image_url} className="info-image"/>
+                    <div className="info-songName">{this.props.relatedSong.song_name}</div>
+                    <div className="info-artistName">
+                        <ArtistTooltip image={artistImage} name={artistName} location={artistLocation} followers={artistFollowers}>
+                            {artistName}
+                        </ArtistTooltip>
```
Comment (mgalfano) [inline client/src/RelatedSong.jsx:23]: Just pointing out another place with the possibly redundant closing tag as above.

**Review by mgalfano — APPROVED** [review]
Looks great, other than my small suggestion (which may be incorrect). Nice job with the tests!

File: client/src/App.jsx (hunk 1 above)
Comment (shiju-an) [reply client/src/App.jsx:45]: actually it breaks up the p tag which is weird!

**Pull request merged**